babel.config.js

```diff
@@ -16,11 +17,7 @@ module.exports = {
     ],
     env: {
         test: {
-            plugins: [
-                '@babel/plugin-transform-runtime',
-                '@babel/plugin-proposal-optional-chaining',
-                '@babel/plugin-proposal-nullish-coalescing-operator',
```

Comment:
We don't need runtime or coalescing anymore?

Reply:
we still have runtime, it just is on the same line now so shows up as a diff